tests/codegen/intrinsics/nontemporal.rs

```diff
@@ -1,13 +1,37 @@
 //@ compile-flags: -O
+//@revisions: with_nontemporal without_nontemporal
+//@[with_nontemporal] compile-flags: --target aarch64-unknown-linux-gnu
+//@[with_nontemporal] needs-llvm-components: aarch64
+//@[without_nontemporal] compile-flags: --target x86_64-unknown-linux-gnu
+//@[without_nontemporal] needs-llvm-components: x86
 
-#![feature(core_intrinsics)]
+// Ensure that we *do* eomit the `!nonteporal` flag on architectures where it
```

Comment:
Nit: sorry this was bugging me 😄 
```suggestion
// Ensure that we *do* omit the `!nontemporal` flag on architectures where it
```